classes/classes/PlayerAppearance.as

```diff
@@ -3,10 +3,46 @@ package classes
 	import classes.*; 
 	import classes.GlobalFlags.*;
 	
-	public class PlayerAppearance extends BaseContent
-	{
+	public class PlayerAppearance extends BaseContent {
+
```

Comment:
Coding guidelines usually say, that the opening brace of class, function and method definitions should be in a new line with some exemtions, like the empty constructor below and short getters and setters. btw. nvm. I'm planning to file a PR including PlayerAppearance later.